.github/workflows/pytest.yml

```diff
@@ -0,0 +1,40 @@
+# Install Python dependencies and run unit tests.
+#
+# Note: These tests should run quickly; we set a timeout to
+# 60 minutes to catch possible hang-ups, should they occur.
+
+name: JMSLab Unit Tests
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 60
```

Comment:
```suggestion
    timeout-minutes: 15
```

Reply:
Per meeting: Fine if this requires economizing on some testing. If there are tests that are valuable but difficult to speed up, we can leave those commented out and plan to incorporate them at major releases only. Since we'll be running tests frequently, it's good if the main testing framework is pretty fast.

Thanks!